root/release/edit/tracklist.tt

```diff
@@ -108,6 +108,7 @@
 
   <!-- ko if: searchResults() && searchResults().length > 0 -->
   <p>[% l('Select a medium from the search results below and then click "Add Medium".') %]</p>
+  <p>[% l('The tracklist for the medium (including the track/recording associations) will be loaded as a medium on this release.') %]</p>
```

Comment:
It should be made more clear that other medium’s attributes won’t be copied:
```suggestion
  <p>[% l('Only the tracklist for the medium (including the track/recording associations) will be loaded as a medium on this release.') %]</p>
```

Reply:
Modified the comment somewhat, let me know how it sounds.

Reply:
SGTM